docs/source/guides/advanced_custom_primitives.ipynb

```diff
@@ -18,30 +15,25 @@
    "source": [
     "from featuretools.primitives import TransformPrimitive\n",
     "from featuretools.tests.testing_utils import make_ecommerce_entityset\n",
-    "from featuretools.variable_types import DatetimeTimeIndex, NaturalLanguage, Numeric\n",
+    "from woodwork.column_schema import ColumnSchema\n",
+    "from woodwork.logical_types import Datetime, NaturalLanguage\n",
     "import featuretools as ft\n",
     "import numpy as np\n",
     "import re"
    ]
   },
   {
-   "cell_type": "raw",
-   "metadata": {
-    "raw_mimetype": "text/restructuredtext"
-   },
+   "cell_type": "markdown",
+   "metadata": {},
    "source": [
-    "Primitives With Additional Arguments\n",
-    "====================================\n",
+    "## Primitives with Additional Arguments\n",
     "\n",
     "Some features require more advanced calculations than others. Advanced features usually entail additional arguments to help output the desired value. With custom primitives, you can use primitive arguments to help you create advanced features.\n",
     "\n",
-    "\n",
-    "String Count Example\n",
-    "~~~~~~~~~~~~~~~~~~~~\n",
-    "\n",
+    "### String Count Example\n",
     "In this example, you will learn how to make custom primitives that take in additional arguments. You will create a primitive to count the number of times a specific string value occurs inside a text.\n",
     "\n",
-    "First, derive a new transform primitive class using :class:`TransformPrimitive` as a base. The primitive will take in a text column as the input and return a numeric column as the output, so set the input type as :class:`NaturalLanguage` and the return type as :class:`Numeric`. The specific string value is the additional argument, so define it as a *keyword* argument inside :meth:`__init__`. Then, override :meth:`get_function` to return a primitive function that will calculate the feature."
+    "First, derive a new transform primitive class using `TransformPrimitive` as a base. The primitive will take in a text column as the input and return a numeric column as the output, so set the input type to a Woodwork column with logical type `NaturalLanguage` and the return type to a Woodwork column with the semantic tag `'numeric'`. The specific string value is the additional argument, so define it as a *keyword* argument inside `__init__`. Then, override `get_function` to return a primitive function that will calculate the feature."
```

Comment:
Should I add something here referencing back to the Woodwork typing guide, or is having a link in the Primitives page under Getting Started enough?

Reply:
I'm thinking the appropriate guide to link to would be the one being created for this issue: https://github.com/alteryx/featuretools/issues/1538

@tamargrey What do you think?

Reply:
I think I agree. Currently the primitives guide doesn't actually go into what a `ColumnSchema` is, so I'm thinking that the woodwork types guide will be a good place to link for this

Reply:
(also, it will probably be worth going through all our guides once we're done and making sure that all the links point to where they're supposed to, since we're talking about linking to things that are also in development)